test/test.py

```diff
@@ -252,25 +255,68 @@ def test_upload_to_cloud_from_s3(self):
         #     "data-bundles-examples/import/10x/pbmc8k/bundles/bundle1/sample.json"
         # ]
         # self.assertEqual(len(uuids), len(names))
-        # print(uuids)
-        # print(names)
         # assert_list_items_equal = (self.assertCountEqual if six.PY3
         #                            else self.assertItemsEqual)
         # assert_list_items_equal(names, out)
 
-    def test_upload_files(self):
-        pass
+    def _reload_api(self):
+        """
+        Ensure that when we reload api, all required methods are present.
+
+        Just reloading api package isn't enough, because the reload method will look
+        first for the compiled python files to load from. This became an issue
+        because api.upload and api.download were not being reloaded (only in Python2 -
+        apparently Python3 reload handles this problem). This method deletes
+        the compiled api init file if it exists before reloading.
+        """
+        pkg_home_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        api_init_compiled_path = os.path.join(pkg_home_dir, "hca", "api", "__init__.pyc")
+        try:
+            os.remove(api_init_compiled_path)
+        except IOError:
+            pass
```

Comment:
Take a look at getting rid of dynamic in-process codegen - split tests into codegen, then separately run test suite